pipelines/controller/src/main/resources/templates/index.html

```diff
@@ -0,0 +1,248 @@
+<html>
+<head>
+
+  <title>
+    FHIR Analytics Pipelines Control Panel
+  </title>
+
+  <link rel="stylesheet" type="text/css" href="../static/style.css"
+        th:href="@{/style.css}">
+  <script>
+    function changeVisibility(elemId, makeVisible) {
+      var elem = document.getElementById(elemId);
+      if (makeVisible) {
+        elem.style.display = 'block';
+      } else {
+        elem.style.display = 'none';
+      }
+    }
+
+    function runPipeline(isFull) {
+      console.log('Trying to start pipeline; isFull: ' + isFull);
+      var xhr = new XMLHttpRequest();
+      xhr.open("POST", '/run', true);
+      xhr.onreadystatechange = function() {
+        if (this.readyState === XMLHttpRequest.DONE && this.status === 200) {
+          console.log('Submit succeeded.');
+          changeVisibility('pipelineSubmit', true);
+          setTimeout(function() {
+            // Reload the page after 2 seconds
```

Comment:
why do we need this timeout?

Reply:
Just for the user to see the new message (note the `changeVisibility` above).